include/zephyr/mgmt/ec_host_cmd/ec_host_cmd.h

```diff
@@ -282,6 +282,31 @@ const struct ec_host_cmd *ec_host_cmd_get_hc(void);
 FUNC_NORETURN void ec_host_cmd_task(void);
 #endif
 
+#ifdef CONFIG_EC_HOST_CMD_IN_PROGRESS_STATUS
```

Comment:
Note - the commit message has a typo. Delete extra "lat"

"Add a config to save the final result of a host command that has"

Reply:
Done.